src/pages/Admin/Details/Form/DetailsForm.tsx

```diff
@@ -112,38 +114,44 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ venue }) => {
         return;
       }
 
-      if (venueId) {
-        const updatedVenue = {
-          ...vals,
-          id: venueId,
-          worldId: venue?.worldId ?? "",
-          parentId: values.parentId,
-        };
-
-        await updateVenue_v2(updatedVenue, user);
-
-        history.push(adminWorldSpacesUrl(venue?.worldId));
-      } else {
-        const newVenue = {
-          ...vals,
-          id: createUrlSafeName(vals.name),
-          worldId: worldId ?? "",
-          parentId: values.parentId ?? "",
-        };
-
-        await createVenue_v2(newVenue, user);
-
-        history.push(adminWorldSpacesUrl(worldId));
+      try {
+        if (venueId) {
+          const updatedVenue = {
+            ...vals,
+            id: venueId,
+            worldId: venue?.worldId ?? "",
+            parentId: values.parentId,
+          };
+
+          await updateVenue_v2(updatedVenue, user);
+
+          history.push(adminWorldSpacesUrl(world?.slug));
+        } else {
+          const newVenue = {
+            ...vals,
+            id: createUrlSafeName(vals.name),
+            worldId: worldId ?? "",
+            parentId: values.parentId ?? "",
+          };
+
+          await createVenue_v2(newVenue, user);
+
+          history.push(adminWorldSpacesUrl(world?.slug));
+        }
+      } catch (e) {
+        console.error(e);
```

Comment:
This will break `const [{ error: submitError,` and really there is no need to log it here since it is the job of these lines https://github.com/sparkletown/sparkle/pull/2474/files#diff-4e9f5a0d22dd89662d1eb6e2d72a265855179bf67c408c5c350292dc9eebef24R355-R356

and an improvement coming along the way with https://github.com/sparkletown/sparkle/pull/2503

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2474/commits/810959fc143f367fa1c4c139201437a894fc3b54